packages/react-core/src/components/DataList/DataList.tsx

```diff
@@ -33,9 +34,10 @@ export const DataList: React.FunctionComponent<DataListProps> = ({
   'aria-label': ariaLabel,
   selectedDataListItemId = '',
   onSelectDataListItem,
+  isCompact = false,
   ...props
 }: DataListProps) => {
-  const isSelectable = !isUndefined(onSelectDataListItem);
+  const isSelectable = onSelectDataListItem !== undefined;
```

Comment:
Why this change? More curious than anything. Using lodash seems like the pattern we follow through patternfly. Is there a benefit for this change?

Reply:
We do not list lodash as a dependency for react-core, and the few calls we make to it are for simple helpers that we can implement ourselves (I literally copied their implementation of `isUndefined` which is just `x === undefined`). The benefits are smaller bundle sizes, making #3836 easier, and one less dependency to track.